packages/runtime/src/runtime/hooks/useEnsembleAction.tsx

```diff
@@ -474,6 +476,16 @@ export const useUploadFiles: EnsembleActionHook<UploadFilesAction> = (
   return { callback };
 };
 
+export const useNavigateBack: EnsembleActionHook<NavigateBackAction> = () => {
+  const navigate = useNavigate();
+
+  const callback = useCallback(() => {
+    navigate(-1);
```

Comment:
You can use the same function between the hook and execute code API. That's the ultimate goal is to share the same code between these.